utils.py

```diff
@@ -453,3 +454,38 @@ def get_all_language_codes_and_names():
 def unescape_encoded_uri_component(escaped_string):
     """Unescape a string that is encoded with encodeURIComponent."""
     return urllib.unquote(escaped_string).decode('utf-8')
+
+
+CACHE_SLUG = None
+def get_cache_slug():
```

Comment:
There are two different places where we use this cache slug. One is in the html and the other is for file paths on disk and they require different ways of interpolating the urls and hence different methods. So, if we combine it with `get_asset_dir_prefix` then we need to again read the cache slug from file in test_utils `get_cache_slug_filepath()`.